pyshelf/metadata/manager.py

```diff
@@ -0,0 +1,122 @@
+from pyshelf.metadata.wrapper import Wrapper
+from pyshelf.metadata.result import Result
+from pyshelf.metadata.error_code import ErrorCode
+import copy
+
+
+class Manager(object):
+    """
```

Comment:
Like this doc block. Good thing to note. It would be super confusing if you were new to this and didn't understand wtf was going on with metadata storage.